card/lib/card/cache.rb

```diff
@@ -25,7 +25,7 @@ class Cache
     class << self
       def [] klass
         raise "nil klass" if klass.nil?
-        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Rails.cache)
+        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Cardio.cache)
```

Comment:
It is supposed to work.  If it doesn't I'll have to fix it.  It may work correctly this way on this branch.